packages/instantsearch.js/src/widgets/sort-by/sort-by.tsx

```diff
@@ -94,6 +94,7 @@ const renderer =
           currentValue={currentRefinement}
           options={options}
           setValue={refine}
+          ariaLabel="Sort results by"
```

Comment:
technically this might not necessarily fit all content allowed in the values, but good enough I think